idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -501,15 +501,26 @@ def check_s3_file_presence(s3_path):
       return 0
 
 # job functions
-def chunk_input(input_files_basenames, chunk_nlines, part_suffix):
+def chunk_input(input_files_basenames, chunk_nlines, chunksize):
```

Comment:
Btw I tested this function in a standalone python interpreter.   Caught one bug, fixed.